stubs/peewee/peewee.pyi

```diff
@@ -880,8 +882,12 @@ class SqliteDatabase(Database):
     def aggregate(self, name: Incomplete | None = ..., num_params: int = ...): ...
     def register_collation(self, fn, name: Incomplete | None = ...): ...
     def collation(self, name: Incomplete | None = ...): ...
-    def register_function(self, fn, name: Incomplete | None = ..., num_params: int = ...) -> None: ...
-    def func(self, name: Incomplete | None = ..., num_params: int = ...): ...
+    def register_function(
+        self, fn, name: Incomplete | None = ..., num_params: int = ..., deterministic: bool | None = None
+    ) -> None: ...
+    def func(
+        self, name: Incomplete | None = ..., num_params: int = ..., deterministic: bool | None = None
+    ) -> Callable[[_F], _F]: ...
```

Comment:
```suggestion
    def register_function(
        self, fn, name: str | None = None, num_params: int = -1, deterministic: bool | None = None
    ) -> None: ...
    def func(
        self, name: str | None = None, num_params: int = -1, deterministic: bool | None = None
    ) -> Callable[[_F], _F]: ...
```

Reply:
`name` is passed to `create_function` in `sqlite` which has `str` annotation 👍